### PR TITLE
Typescript

### PR DESCRIPTION
**Pull request by standvpmnt**

### CODE REVIEW

File: Typescript/TypeBasic.md [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+
+string represents string values like "Hello, world"
+`let title: string = "Web Developer"`;
+
+**Boolean**
+
+boolean is for the two values true and false
+`let pending: boolean`;
+`pending = true`;
+
```
Comment (standvpmnt) [inline Typescript/TypeBasic.md:50]: Please add other 3 types as well, undefined, null and symbol. Also mention during the discussion about bigint etc.

File: Typescript/TypeBasic.md [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+}
+```
+
+we can use optional properties
+
+```
+type personObj = {
+name:string;
+age:number,
+mobile? = number
```
Comment (standvpmnt) [inline Typescript/TypeBasic.md:84]: : instead of =

File: Typescript/classes/second.ts [hunk 3]
```diff
@@ -0,0 +1,54 @@
+// implements with interface
+interface Human {
+  name: string;
+  contactNumber: string;
+}
+
```
Comment (standvpmnt) [inline Typescript/classes/second.ts:6]: interface with extending another interface

**Comment by Shiv266** [comment]
@standvpmnt sir i have made changes, please review and merge pr

**Comment by standvpmnt** [comment]
@Shiv266 I think you will have to review and merge this PR, as I am marked as the author on this one

**Pull request merged**